jwst/regtest/test_nircam_tsgrism.py

```diff
@@ -88,36 +87,12 @@ def test_nircam_tsgrism_stage3_x1dints(run_pipelines, fitsdiff_default_kwargs):
 @pytest.mark.bigdata
 def test_nircam_tsgrism_stage3_whtlt(run_pipelines):
     rtdata = run_pipelines
-    rtdata.input = "jw00721-o012_20191119t043909_tso3_001_asn.json"
-    rtdata.output = "jw00721-o012_t004_nircam_f444w-grismr-subgrism256_whtlt.ecsv"
-    rtdata.get_truth("truth/test_nircam_tsgrism_stages/jw00721-o012_t004_nircam_f444w-grismr-subgrism256_whtlt.ecsv")
+    rtdata.input = "jw01366-o002_20230107t004627_tso3_00001_asn.json"
+    rtdata.output = "jw01366-o002_t001_nircam_f322w2-grismr-subgrism256_whtlt.ecsv"
+    rtdata.get_truth("truth/test_nircam_tsgrism_stages/jw01366-o002_t001_nircam_f322w2-grismr-subgrism256_whtlt.ecsv")
 
     table = Table.read(rtdata.output)
     table_truth = Table.read(rtdata.truth)
 
     # setdiff returns a table of length zero if there is no difference
     assert len(setdiff(table, table_truth)) == 0
-
-
-@pytest.mark.bigdata
-def test_nircam_setpointing_tsgrism(rtdata, fitsdiff_default_kwargs):
```

Comment:
can you explain how/why this was "quietly broken"?

Reply:
PR 6993 was a significant change to how `set_telescope_pointing` worked.  It used to allow a sqlite database input, which is how the nircam tsgrism and tsimg tests were set up to work; after that PR, it no longer did, but the regression tests were not updated to take the newer-style input.

This was probably missed because the tests also included a catch for cases where the test could not be run for external reasons (pysiaf not installed, engineering database not available).  In case of any ValueError, the test was skipped.  After the change to `set_telescope_pointing`, the code always throws a ValueError, so the test is always quietly skipped.

Reply:
cool thanks for the info.  A good reminder to be very specific about what kinds of errors you want to catch